kotlin-eclipse-ui/src/org/jetbrains/kotlin/ui/editors/templates/KotlinDocumentTemplateContext.java

```diff
@@ -44,8 +47,13 @@ public TemplateBuffer evaluate(Template template) throws BadLocationException, T
         
         KotlinTemplateFormatter templateFormatter = new KotlinTemplateFormatter();
         
-        IJavaProject javaProject = JavaCore.create(EditorUtil.getFile(editor).getProject());
-        templateFormatter.format(templateBuffer, getLineIndentation(), javaProject);
+        IFile file = EditorUtil.getFile(editor);
+        if (file != null) {
```

Comment:
Could you please give an example when file here might become `null`? 
As template processing is part of completion processing I think that we have checked file for notnull earlier